Documentation/ceph-nfs-crd.md

```diff
@@ -66,6 +70,20 @@ spec:
     priorityClassName:
 ```
 
+ Enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running the following command in the toolbox container:
+
+[For single NFS-GANESHA CLUSTERS](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)  
```

Comment:
```suggestion
[For single NFS-GANESHA cluster](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)  
```

Reply:
@leseb not sure if we need a ceph link here?

Reply:
I'm fine with more doc anyways :)